crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1725,9 +1725,73 @@ impl PgManager {
                 },
             ));
         }
-
         Ok(Some(connection))
     }
+
+    pub(crate) async fn fetch_dynamic_field_object(
+        &self,
+        address: SuiAddress,
+        name: DynamicFieldName,
+    ) -> Result<Option<DynamicField>, Error> {
+        let type_tag =
+            TypeTag::from_str(&name.type_).map_err(|e| Error::Internal(format!("{e}")))?;
```

Comment:
Clippy should tell you this as well, but:

```suggestion
            TypeTag::from_str(&name.type_).map_err(|e| Error::Internal(e.to_string()))?;
```